data/nfv/conf/00_common.conf

```diff
@@ -0,0 +1,154 @@
+# Copyright 2015-2017 Intel Corporation.
```

Comment:
is this file fully copied and not changed from some repo? maybe better copy it at runtime from this place?

Reply:
These bunch of files are modified from the original according to our needs. I am actually overwriting the originals which are already there when cloning the repo